### PR TITLE
Combine related paragraphs and add more details for implementations

### PR DESCRIPTION
**Pull request by JoshBrudnak**

### CODE REVIEW

**Comment by JoshBrudnak** [comment]
Fixes: #372

File: src/items/implementations.md [hunk 1]
```diff
@@ -1,32 +1,31 @@
...
-the associate item's path component.
-
-Inherent implementations cannot contain associated type aliases.
+the associate item's path component. Inherent implementations cannot contain
+associated type aliases.
 
-A type can have multiple inherent implementations.
-
-The implementing type must be defined within the same crate.
+A type can also have multiple inherent implementations. Although the
```
Comment (Havvy) [inline src/items/implementations.md]: The "Although" used to join here feels very artificial.

**Comment by Havvy** [comment]
LGTM. I didn't realize when I did all the changes around here I made the paragraphs so separated like that. My only quibble is that the "Although" joiner really is joining two disparate pieces of information. But that's ultimately a quibble.

Since I've touched this page so much, I'd rather the r+ come from another person before merging.

File: src/items/implementations.md (hunk 1 above)
Comment (JoshBrudnak) [reply src/items/implementations.md]: Yes, what if I use "but" instead of "although"

Comment (Havvy) [reply src/items/implementations.md]: The "but" implies an exception on the previous statement. An "and" would do well if it cannot read well as two sentences.

Comment (JoshBrudnak) [reply src/items/implementations.md]: @Havvy  Does this sound better: "An implementing type must be defined within the same crate as the original type definition."

**Comment by JoshBrudnak** [comment]
@Havvy I separated that second clause into "An implementing type must be defined within the same crate as the original type definition.".

**Comment by JoshBrudnak** [comment]
@alercah can you review my changes? Thanks :100:

**Comment by Havvy** [comment]
@alercah has been busy, and @matthewjasper has given their seal of approval. So gonna merge it.

💟 Thanks!

**Pull request merged**